components/NavigationDrawer/NavigationDrawerListItem/index.tsx

```diff
@@ -1,30 +1,31 @@
 import ListItemText from '@mui/material/ListItemText'
 import ListItemIcon from '@mui/material/ListItemIcon'
 import ListItemButton from '@mui/material/ListItemButton'
+import Link from 'next/link'
 // props for the list
 interface ListProps {
   text: string
   icon?: React.ReactNode
   collapsable?: boolean
+  route: string
 }
 
 function NavigationDrawerListItem({
   collapsable = false,
   icon,
   text,
+  route,
 }: ListProps) {
```

Comment:
add below this line a call to the next router hook `useRouter`

```ts
const router = useRouter();
```

`useRouter` can be imported using the following syntax:
```js
import { useRouter } from 'next/router
```